gemet/thesaurus/edit_views.py

```diff
@@ -77,6 +77,14 @@ def _get_response(self, data, status, status_code):
         return response
 
 
+class FormatFormErrorsMixin(object):
+
+    def _get_form_errors(self, errors):
+
+        return reduce(lambda x, y: x + ' ' + y,
+                      [error[1][0] for error in errors.items()])
```

Comment:
This is not very readable. `errors` is a `dict`, you're iterating over the keys, and then concatenating the elements at the position `[1][0]` in each key? Why? One should understand easily the structure of each element in `errors` and the purpose of these operations.